src/hooks/misc/remove_player_from_sender_list.cpp

```diff
@@ -0,0 +1,11 @@
+#include "hooking.hpp"
+#include "pointers.hpp"
+
+namespace big
+{
+	bool hooks::remove_player_from_sender_list(void* list, uint64_t rockstar_id)
+	{
+		g_pointers->m_gta.m_remove_player_from_sender_list(list, &rockstar_id);
```

Comment:
Incorrectly typed parameter here. rockstar_id should be a uint64_t * since we type m_rockstar_id coming from the player info class as uint64_t, but it is clearly a pointer passed to it, for an unbeknownst reason.